pnpm-lock.yaml

```diff
@@ -8960,7 +8964,7 @@ packages:
     dev: true
 
   /html-escaper/2.0.2:
-    resolution: {integrity: sha512-H2iMtd0I4Mt5eYiapRdIDjp+XzelXQ0tFE4JS7YFwFevXXMmOp9myNrUvCg0D6ws8iqkRPBfKHgbwig1SmlLfg==}
+    resolution: {integrity: sha1-39YAJ9o2o238viNiYsAKWCJoFFM=}
```

Comment:
why were these integrities downgraded? do you use some non-standard registry?

Reply:
I am totally have no idea about this. It feels like a long term issue existing in npm registry.